pilot/pkg/xds/sds.go

```diff
@@ -343,6 +343,10 @@ func toEnvoyTLSSecret(name string, certInfo *credscontroller.CertInfo, proxy *mo
 				},
 			},
 		})
+		fallback := false
+		if crypto.GetFallback() != nil {
+			fallback = crypto.GetFallback().Value
```

Comment:
nit: you can just do crypto.GetFallback().GetValue(). It is nil-safe and will return `false` if Fallback is nil